Library/Formula/kjell.rb

```diff
@@ -15,15 +17,14 @@ def install
   end
 
   test do
-    require 'open3'
-    Open3.popen3("#{bin}/kjell") do |stdin, stdout, _|
+    require "open3"
+    Open3.popen3("#{bin}/kjell") do |stdin, _, _|
```

Comment:
Can't pipe commands into `kjell`, it only allows you to open the interactive shell and then input.